package.json

```diff
@@ -115,6 +116,7 @@
     "flag-icon-css": "3.3.0",
     "flux": "3.1.3",
     "fontfaceobserver": "2.1.0",
+    "format-currency": "file:./packages/format-currency",
```

Comment:
Oops, leftover entry here -- we don't need this anymore.

Reply:
(I think this is also what's causing CircleCI errors)

Reply:
Indeed:

```
npm ERR! Missing: format-currency@file:./packages/format-currency
```

```suggestion
```